scripts/start.js

```diff
@@ -347,6 +357,10 @@ function getUserInfo () {
     defaultInput: DEFAULT_DOMAIN
   });
 
+  var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
I believe this is unnecessary almost every time. Provided we have an auth token (based on credentials entered above) we can access `api/users`, pick the one with matching username and domain `internal-authz` and offer it as an offered default, or just use it and let developers to edit the script in case someone wants to use  user with different domain.

AFAIK there is no clear mapping between domain entered by user (`internal`) and domain show in API (`internal-authz`).

Sample of API output:

    <user href="/ovirt-engine/api/domains/696E7465726E616C2D617574687A/users/64393165343532662D326330312D343266632D613365632D636537383930373430373862" id="64393165343532662D326330312D343266632D613365632D636537383930373430373862">
        <name></name>
        <department></department>
        <email></email>
        <last_name></last_name>
        <namespace>*</namespace>
        <principal>ovirt02</principal>
        <user_name>ovirt02@internal-authz</user_name>
        <domain href="/ovirt-engine/api/domains/696E7465726E616C2D617574687A" id="696E7465726E616C2D617574687A">
            <name>internal-authz</name>
        </domain>
        <groups/>
    </user>

Of course, it is not necessary to be part of this pull, just an idea.